docs/appnotes/0004-tams-for-data.md

```diff
@@ -0,0 +1,60 @@
+# When TAMS is a good fit for non-media data. And when it’s not
+
+## Abstract
+
+TAMS places few restrictions on the type of content stored within it.
+It has been developed primarily with audio and video in mind.
+But there are other types of timeline-based content one might wish to store in their media workflows.
+When is it appropriate to store these other types of content in TAMS?
+
+## Considerations
+
+### Indexing of content
+
+TAMS indexes content based on a small number of specific types of metadata.
+Content may be queried based on these types of metadata (e.g. Flow/Source ID, tags, time-ranges etc).
+These metadata apply to the Source/Flow as a whole.
+It is not possible to query the data itself, or metadata as it applies to a portion of the Source/Flow.
+As such, TAMS is usually not appropriate for the storing of timeline-based metadata.
+
+### Segment size
+
+TAMS may be a good fit for querying some types of content.
+But it might not be an efficient storage mechanism.
+Typically, the content being stored should be larger than the metadata stored alongside it.
+
+## Examples
+
+### Timed Text
+
+Timed text, such as closed captions, is relatively low data rate.
+But it is often used alongside audio and video, using similar workflows.
+Existing live workflows in television broadcast facilities may make use of [SMPTE 2110-43](https://ieeexplore.ieee.org/document/9521125)/[RFC8759](https://www.rfc-editor.org/rfc/rfc8759.html) to carry small independently XML documents containing the captions.
+The usage patterns, recommended frequency, and likely size of these documents lend themselves well to storage in a TAMS store.
```

Comment:
It might be worth spelling this out a bit more completely
```suggestion
The usage patterns, recommended frequency, and likely size of these documents lend themselves well to storage in a TAMS store, however care should be taken to balance the flexibility of shorter segments (and smaller XML documents) against the increased metadata storage overhead.
```